db/migrations/000001_init.up.sql

```diff
@@ -0,0 +1,73 @@
+BEGIN;
+
+CREATE TABLE "users" (
+                         "id" uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+                         "auth0_id" varchar(40) UNIQUE NOT NULL,
+                         "name" varchar(20) NOT NULL,
+                         "email" varchar(100) UNIQUE NOT NULL,
+                         "picture" varchar(255),
+                         "created_at" timestamptz NOT NULL DEFAULT 'now()',
+                         "updated_at" timestamptz NOT NULL DEFAULT 'now()'
+);
+
+CREATE TABLE "todos" (
+                         "id" uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+                         "title" varchar(80) NOT NULL,
+                         "description" varchar,
+                         "done" bool NOT NULL DEFAULT false,
+                         "priority" varchar(20) NOT NULL DEFAULT 'mid',
+                         "due_date" timestamptz,
+                         "project_id" uuid NOT NULL,
+                         "user_id" uuid NOT NULL,
+                         "created_at" timestamptz NOT NULL DEFAULT 'now()',
+                         "updated_at" timestamptz NOT NULL DEFAULT 'now()'
+);
+
+CREATE TABLE "priorities" (
+                              "priority" varchar(20) PRIMARY KEY,
+                              "created_at" timestamptz NOT NULL DEFAULT 'now()',
+                              "updated_at" timestamptz NOT NULL DEFAULT 'now()'
+);
+
+
+CREATE TABLE "projects" (
+                            "id" uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+                            "owner_id" uuid NOT NULL,
```

Comment:
どのテーブルに関連づいているのかをカラムの命名に込めるとコードの可読性が上がりそうかなと思いました

```suggestion
                            "owner_user_id" uuid NOT NULL,
```

Reply:
そのテクニックは知りませんでした!
個人的にはownerという命名からuserの一種であることは推測できそうな気がする(コードを見ればなおさら)ので若干冗長な印象を受けてしまったのですが、カラムの命名は極力具体的な方がベストなのでしょうか？

Reply:
> 個人的にはownerという命名からuserの一種であることは推測できそうな気がする

まぁ確かにそうですね...であればこのままで OK です！

> カラムの命名は極力具体的な方がベストなのでしょうか？

そうですね。具体的なほうがわかりやすいと思ってますが、冗長な命名は自分も好きじゃないです